tools/c7n_mailer/c7n_mailer/utils.py

```diff
@@ -284,6 +284,37 @@ def resource_format(resource, resource_type):
         return "QueueURL: %s QueueArn: %s " % (
             resource['QueueUrl'],
             resource['QueueArn'])
+    elif resource_type == "efs":
+        return "name: %s  id: %s  state: %s" % (
+            resource['Name'],
+            resource['FileSystemId'],
+            resource['LifeCycleState']
+        )
+    elif resource_type == "network-addr":
+        return "ip: %s  id: %s  scope: %s" % (
+            resource['PublicIp'],
+            resource['AllocationId'],
+            resource['Domain']
+        )
+    elif resource_type == "route-table":
+        return "id: %s  vpc: %s" % (
+            resource['RouteTableId'],
+            resource['VpcId']
+        )
+    elif resource_type == "app-elb":
+        return "name: %s  zones: %s  scheme: %s" % (
+            resource['LoadBalancerName'],
```

Comment:
nit: so a given account/region can have overlapped app load balancer names afaicr, it might be better to have LoadsBalancerArn here.

Reply:
ARN would be nice. It's up to you to if you think we should change it. I've always been a fan of Names as they are more user readable...especially if they are required and not from the "Name" tag.
But I guess ARN does provide additional details like account number...

Reply:
the issue with a name in that context, is if you have multiple alb resources with the same name, it actually has potential to cause more confusion.

Reply:
ah yes.. forgot about the region overlap. okay I will use the ARN as it provides more information upfront